website/src/content/deployment/runtime-configuration.mdx

```diff
@@ -63,6 +63,18 @@ An example configuration for local development:
 }
 ```
 
+Lastly, any envrionment configuration JSON file can contain placeholders for environment variables. This can be useful to avoid duplication between various `env.json` files for multiple different environments. Placeholders are specified with an expansion like syntax `${}` while additionally prefixing the environment variable name with `env:`.
```

Comment:
Public docs.

Reply:
I like that we are using a placeholder instead of multiple env.json files which can get out of sync with each other more easily than the approach you picked here. nice!